testnet/stacks-node/src/node.rs

```diff
@@ -761,6 +761,30 @@ impl Node {
 
             parent_consensus_hash
         };
+
+        // get previous burn block stats
+        let (parent_burn_block_hash, parent_burn_block_height, parent_burn_block_timestamp) =
+            if anchored_block.is_first_mined() {
+                (BurnchainHeaderHash([0; 32]), 0, 0)
+            } else {
+                match SortitionDB::get_block_snapshot_consensus(db.conn(), &parent_consensus_hash)
+                    .unwrap()
+                {
+                    Some(sn) => (
+                        sn.burn_header_hash,
+                        sn.block_height as u32,
+                        sn.burn_header_timestamp,
+                    ),
+                    None => {
+                        // shouldn't happen
+                        panic!(
```

Comment:
Same here -- this `panic!` is reachable under normal circumstances, like a PoX reorg.